bugbot/rules/perfalert_inactive_regression.py

```diff
@@ -0,0 +1,155 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import collections
+
+from libmozdata.bugzilla import Bugzilla
+
+from bugbot import logger, utils
+from bugbot.bzcleaner import BzCleaner
+from bugbot.user_activity import UserActivity, UserStatus
+
+
+class PerfAlertInactiveRegression(BzCleaner):
+    def __init__(self, nweeks=1):
+        super().__init__()
+        self.nweeks = nweeks
+        self.extra_ni = {}
+        self.private_regressor_ids: set[str] = set()
+
+    def description(self):
+        return f"PerfAlert regressions nag with {self.nweeks} week of inactivity"
+
+    def handle_bug(self, bug, data):
+        if len(bug["regressed_by"]) != 1:
+            # either we don't have access to the regressor,
+            # or there's more than one, either way leave things alone
+            return
+
+        data[str(bug["id"])] = {
+            "regressor_id": bug["regressed_by"][0],
+        }
+
+        return bug
+
+    def get_bz_params(self, date):
+        start_date, _ = self.get_dates(date)
+
+        fields = [
+            "id",
+            "regressed_by",
+        ]
+
+        # Find all bugs with regressed_by information which were open after start_date or
+        # whose regressed_by field was set after start_date.
+        params = {
+            "include_fields": fields,
+            "f3": "creation_ts",
+            "o3": "greaterthan",
+            "v3": "2024-10-01T00:00:00Z",
```

Comment:
I think this is to avoid noise by ignoring old cases. Could you please add a clarifying comment? That would be helpful for future us.

Reply:
That's correct, I've added a comment for this.